packages/element-library/src/video/display.js

```diff
@@ -46,6 +46,7 @@ function VideoDisplay({
     focalX,
     focalY,
     loop,
+    volume = 1.0,
```

Comment:
We don't need this default value since `1.0` is the default anyway in HTML.

Reply:
I see the default values are still there, what's the rationale for that? We should be able to just omit this unless I am missing something.

Reply:
@spacedmonkey why was this comment resolved? It‘s still valid and I‘d appreciate a reply.

Reply:
I changed it from 1.0 to 1 as requested.

Reply:
My request was to remove it entirely.